postr/config.py

```diff
@@ -18,6 +18,8 @@
     'Twitter': {},
     'Reddit': {
         'subreddit': '',
+        'client_id': 'AZQxN0WW9txW3g',
```

Comment:
Don't commit any auth tokens, ever!

 These will go in the postr_config.ini file. You're retrieving them right , but they should be in the reddit section of the ini file, you can add them by pasting them in this format:

client_id = AZQxn.....

or by adding them using python to make sure it's formatted right (uneeded)

but never committed!